jib-core/src/main/java/com/google/cloud/tools/jib/registry/InsecureRegistryException.java

```diff
@@ -24,6 +24,6 @@
 public class InsecureRegistryException extends RegistryException {
 
   InsecureRegistryException(URL insecureUrl) {
-    super("Only secure connections are allowed, but tried to reach URL " + insecureUrl);
+    super("Only secure connections are allowed, but failed to verify the server at " + insecureUrl);
```

Comment:
The phrasing here is a bit awkward, maybe something like "Failed to verify the server at `<url>` because only secure connections are allowed"